content/en/serverless/google_cloud_run/_index.md

```diff
@@ -2,43 +2,68 @@
 title: Google Cloud Run
 kind: documentation
 further_reading:
-    - link: 'https://www.datadoghq.com/blog/collect-traces-logs-from-cloud-run-with-datadog/'
-      tag: 'Blog'
-      text: 'Collect traces, logs, and custom metrics from GCR services'
+
+- link: 'https://www.datadoghq.com/blog/collect-traces-logs-from-cloud-run-with-datadog/'
+  tag: 'Blog'
+  text: 'Collect traces, logs, and custom metrics from GCR services'
+
 ---
 
 ## Overview
-Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog provides monitoring and log collection for Cloud Run through the [GCP integration][1]. Datadog also provides a solution, now in public beta, for instrumenting your Cloud Run applications with a purpose-built Agent to enable tracing, custom metrics, and direct log collection.
+
+Google Cloud Run is a fully managed serverless platform for deploying and scaling container-based applications. Datadog
+provides monitoring and log collection for Cloud Run through the [GCP integration][1]. Datadog also provides a solution,
+now in public beta, for instrumenting your Cloud Run applications with a purpose-built Agent to enable tracing, custom
+metrics, and direct log collection.
 
   <div class="alert alert-warning">This feature is in public beta. You can provide feedback through a <a href="https://forms.gle/HSiDGnTPvDvbzDAQA">feedback form</a>, or through your standard support channels. During the beta period, Cloud Run monitoring and APM tracing are available without a direct cost. Existing APM customers may incur increased span ingestion and volume costs. </div>
 
-## Tracing and custom metrics
-### Build your container
+## Getting started
```

Comment:
After reading through your PR, I think we can further improve the structure to something below that logically reflects the steps people need to follow on the high level:

## Getting started
### Prerequisites
### Deploy a sample application in one click
### Configure the Datadog Agent
#### Using Dockerfile
#### Using Buildpack
### Configure the Datadog API key
### Configure tracing
### Configure custom metrics
### Configure logging
### Deploy to Google Cloud Run

What do you think?

Reply:
I like the idea, but for configure logging/custom metrics there is 1 line of instruction (or it's already set up) and the big titles just drain so much space and make it look longer than it needs.
Also I added `tracing` env in the instruction with the other env variables